tools/oesign/main.c

```diff
@@ -468,11 +474,16 @@ static const char _usage_gen[] =
 static const char _usage_sign[] =
     "Usage: %s sign {--enclave-image | -e} ENCLAVE_IMAGE "
     "{--config-file | -c} CONFIG_FILE {--key-file | -k} KEY_FILE\n"
+    "{--engine ENGINE-NAME -load-path ENGINE-LOAD-PATH-key KEY-ID }\n"
     "\n"
     "Where:\n"
     "    ENCLAVE_IMAGE -- path of an enclave image file\n"
     "    CONFIG_FILE -- configuration file containing enclave properties\n"
     "    KEY_FILE -- private key file used to digitally sign the image\n"
+    "    ENGINE-NAME -- text name of the engine to use, for example 'pkcs-11'"
```

Comment:
`\n` needed here and at the end of each of the ENGINE-LOADPATH and KEY-ID descriptions.

Reply:
added